src/operator/account/MigrateOrgs.tsx

```diff
@@ -0,0 +1,85 @@
+import {
+  AlignItems,
+  Button,
+  ButtonType,
+  ComponentColor,
+  ComponentSize,
+  ComponentStatus,
+  FlexBox,
+  FlexDirection,
+  Form,
+  Input,
+} from '@influxdata/clockface'
+import React, {ChangeEvent, FC, useContext, useState} from 'react'
+import {AccountContext} from '../context/account'
+import MigrateOrgsOverlay from './MigrateOrgsOverlay'
+import {OperatorAccount, getOperatorAccount} from 'src/client/unityRoutes'
+
+const MigrateOrgsTool: FC = () => {
+  const {account, setMigrateOverlayVisible} = useContext(AccountContext)
+  const [toAccountId, setToAccountId] = useState('')
+  const [toAccount, setToAccount] = useState<OperatorAccount>(null)
+
+  const changeToAccountId = (event: ChangeEvent<HTMLInputElement>) => {
+    setToAccountId(event.target.value)
+  }
+
+  const submit = async () => {
+    if (toAccountId === '') {
+      return
+    }
+
+    try {
+      const resp = await getOperatorAccount({accountId: toAccountId})
+      if (resp.status !== 200) {
+        return
```

Comment:
Same - could we add a notification letting the user know there was an error retrieving the `ToAccount`?